### ISSUE TITLE
'This app can't run on your PC' Windows 10

### ISSUE THREAD

**Issue by BlackBank** [Southclaws/sampctl#273]
When i download and install the **sampctl_1.8.36_windows_amd64.tar.gz** version and typing the command **sampctl** i get a Windows 10 error message saying that the app can't be run.

This is the error message i get:

![this-app-cant-run-on-your-pc](https://user-images.githubusercontent.com/5054985/49481758-9cefa280-f82c-11e8-8e39-71a9cb32a756.png)

Did i missed something during the installation?

**Comment by spacemud** [comment]
Are you running the 64 bit version on a 32 bit machine?

**Comment by Southclaws** [comment]
As spacemud said, that (utterly useless) error "message" *sometimes* appears when you run a 64 bit app on a 32 bit machine (of course we have no idea because the error is so vague... /rant)

Try the "i386" version instead of the "amd64" version (weird naming, I know)

**Comment by BlackBank** [comment]
Oh, i forget to mention that i run a 64-bit Windows 10 Pro OS, so amd64 is the right one. ;)

I did try the 'i386' version, but sadly i get the same message as desribed aboved.

**Comment by Southclaws** [comment]
In that case, I have no idea without more information.

**Comment by BigETI** [comment]
Please post your hardware specs. Do you run Windows on a x86 or x64 machine?

**Comment by BlackBank** [comment]
Here they are:
```
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.17134 N/A Build 17134
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
Registered Owner:          Windows User
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 42 Stepping 7 GenuineIntel ~3401 Mhz
Windows Directory:         C:\WINDOWS
System Directory:          C:\WINDOWS\system32
Boot Device:               \Device\HarddiskVolume1
System Locale:             en-gb;Engels (Verenigd Koninkrijk)
Input Locale:              en-us;Engels (Verenigde Staten)
Time Zone:                 (UTC+01:00) Brussel, Kopenhagen, Madrid, Parijs
Total Physical Memory:     16.349 MB
Page File Location(s):     D:\pagefile.sys
Domain:                    WORKGROUP
Logon Server:              \\JORIS-PC
Hotfix(s):                 8 Hotfix(s) Installed.
                           [01]: KB4100347
                           [02]: KB4338853
                           [03]: KB4343669
                           [04]: KB4456655
                           [05]: KB4465663
                           [06]: KB4471331
                           [07]: KB4477029
                           [08]: KB4467682
```
If i miss something, let me know. ;-)

**Comment by Southclaws** [comment]
I can't tell why the error appears from that info, everything looks fine.

Maybe it's some security setting that only allows signed binaries to run. I don't know Windows permissions that well though.

**Comment by BlackBank** [comment]
Seems to be fixed with version **1.8.38**!

**Comment by Southclaws** [comment]
Good to know, thanks for the update!

I still don't know why though, nothing significant changed in 1.8.38 it was very minor changes. So I'm gonna go ahead and assume it was a Windows update or something else weird.
